src/components/BorderlessButton.tsx

```diff
@@ -12,7 +13,12 @@ export type Props = Omit<TouchableProps, 'style'> & {
 export default function BorderlessButton(props: Props) {
   const { style, containerStyle, children, notScaleFont, ...passThroughProps } = props
   return (
-    <Touchable {...passThroughProps} borderless={true} style={containerStyle}>
+    <Touchable
+      {...passThroughProps}
+      borderless={true}
+      style={containerStyle}
+      hitSlop={variables.iconHitslop}
```

Comment:
i considered adding it to the "explore dapps" text only, but seems like a hitslop is always a good idea to make the touch target bigger